### DISCUSSION TITLE
Is there a way to purge app.log periodically?

### DISCUSSION THREAD

**Discussion by meeowgler** [netalertx/NetAlertX#812] [General]
My app.log seems to continuously grow and never purges. Is there a way to purge app.log periodically?

**Comment by jokob-sk** [comment]
Hi @meeowgler ,

You can adjust the log length and cleanup schedule via the Maintenance plugin (`MAINT_LOG_LENGTH` and `MAINT_RUN_SCHD` settings).

J
